irohad/main/CMakeLists.txt

```diff
@@ -63,10 +64,14 @@ target_link_libraries(irohad
     keys_manager
     common
     iroha_conf_loader
+    logger
+    logger_manager
     )
 
 add_library(iroha_conf_loader iroha_conf_loader.cpp)
-target_link_libraries(iroha_conf_loader iroha_conf_literals)
+target_link_libraries(iroha_conf_loader
+  iroha_conf_literals
+  logger_manager)
```

Comment:
minor formatting issues
indentation needs to be increased
according to all other places, closing bracket has to be placed on a new line